rasa/shared/nlu/training_data/features.py

```diff
@@ -27,6 +27,24 @@ def __init__(
         self.origin = origin
         self.attribute = attribute
 
+    def __repr__(self) -> Text:
+        return (
+            f"{self.__class__.__name__}("
+            f"features={self.features}, "
+            f"type={self.type}, "
+            f"origin={self.origin}, "
+            f"attribute={self.attribute})"
+        )
+
+    def __str__(self) -> Text:
+        return (
+            f"{self.__class__.__name__}("
+            f"features.shape={self.features.shape}, "
+            f"type={self.type}, "
+            f"origin={self.origin}, "
+            f"attribute={self.attribute})"
+        )
```

Comment:
Looks like a duplicate to `__str__`. One could just call the other.

Reply:
xD yep

Reply:
Tricked myself. The other one is `__repr__` which shows the full `self.features` while `__str__` omits those values and only prints the shapes instead.